src/Libraries/Directory.Build.props

```diff
@@ -3,7 +3,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreTargetFrameworks)$(ConditionalNet462)</TargetFrameworks>
-    <IsTrimmable Condition="'$(IsTrimmable)' == ''">true</IsTrimmable>
+    <IsTrimmable Condition="'$(IsTrimmable)' == '' and $([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net8.0'))">true</IsTrimmable>
```

Comment:
Just curious, why not trim the other TFMs like net462?

Reply:
Started to get build warnings with the latest SDK about trimming .NET Framework stuff not being a good idea. So this shuts up the warning.